src/labs.txt

```diff
@@ -93,6 +85,18 @@ Output:
 =init
 EOF
 
+h2. Setup Your Name and Email
+
+p. Run the following commands so that git knows your name and email.
```

Comment:
With this change, I'd personally probably reword this to something like 

    Run the following commands so that your current local git repository knows your name and email.

The command wont really make "git" know about it, only "git in this specific local `.git/` location.

Personally, I'd probably keep the global instructions, and flip the note.  Meaning make the note point out how these commands can be run **without** the `--global` flag inside a git repository to change the global config on a per repository basis.

Thoughts?

Reply:
I think @joealba specifically called out needing to use client-specific emails in client repositories, which is why I went for demonstrating the local version and noting the global one. It also lines up with using the defaults and calling out the options separately, which I prefer.

What about, "Run the following commands so that git knows your name and email in this repository."

Reply:
I've updated the wording slightly and rebased. Preview at git.withmojo.com/dw/per-repo-config

Reply:
I think the `--global` config seems most relevant to the current context of setting up default configurations in this doc.  Once that is described, then the repo-specific config could be described as a sidenote or additional scenario.

- Set your _default_ name/email for all your git commits
- Sometimes you may need to override this default and use a different email address for a specific repository.  Maybe it's a personal project repository or it is a client's repository where the client has provided you with an email address under their domain.  To do so...